lib/app/features/account/account.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class AccountPage extends StatelessWidget {
```

Comment:
Please rename it to `AccountScreen` as well as file. 
By name account someone might think that's a model.